src/administration/backup_and_restore.md

```diff
@@ -64,6 +64,9 @@ Get into `psql` for your docker:
 `docker-compose exec postgres psql -U lemmy`
 
 ```sql
+-- Instance
+update instance set domain = replace (domain, 'old_domain', 'new_domain');
```

Comment:
You also need to update these:
- site.actor_id
- site.inbox_url
- site.sidebar

Reply:
Updated